src/nvim/ops.c

```diff
@@ -3644,6 +3644,10 @@ int do_join(size_t count,
    */
   for (t = (linenr_T)count - 1;; t--) {
     cend -= currsize;
+	if(cend == NULL || curr == NULL){
```

Comment:
How is that done? `git rebase <feature branch> -i exec` while in master? Additionally, why is rebase used instead of merging? Is it just preference?